--- configure.ac        2017-02-27 03:16:33.586019372 +0000
+++ configure_rhel.ac   2017-02-27 03:17:07.485996301 +0000
@@ -58,8 +58,14 @@
     *)
       SYSTEM_INCLUDES=""
       SYSTEM_LIBS="-lrt -ldl -pthread"
-      USER_INCLUDES_PATH="/usr/local/include"
-      USER_LIBS_PATH="/usr/local/lib"
+      if [[ -f "/etc/redhat-release" ]]; then
+          USER_INCLUDES_PATH="/usr/include"
+          USER_LIBS_PATH="/usr/lib64"
+      else
+          USER_INCLUDES_PATH="/usr/local/include"
+          USER_LIBS_PATH="/usr/local/lib"
+      fi
+      ;;
 esac
 
 AC_SUBST(CFLAGS)
