--- Core3/MMOCoreORB/build/unix/src/Makefile	2018-04-07 23:52:55.685481833 -0400
+++ Core3/MMOCoreORB/build/unix/src/Makefile_generic_x86-64_build	2018-04-07 23:55:28.170788631 -0400
@@ -1237,7 +1237,7 @@
 top_build_prefix = ../
 top_builddir = ..
 top_srcdir = ../../..
-DEFAULT_FLAGS = $(UNITTEST_DIRECTIVES) -ggdb -Wall -Wextra -Wno-unused -Wno-sign-compare -Wno-multichar -DDEFAULT_DYNAMIC_CAST -Wno-unused-parameter -Wno-overloaded-virtual -Wno-missing-declarations -Werror -std=c++11 -march=native -DCOMPILE_CORE3_TESTS
+DEFAULT_FLAGS = $(UNITTEST_DIRECTIVES) -ggdb -Wall -Wextra -Wno-unused -Wno-sign-compare -Wno-multichar -DDEFAULT_DYNAMIC_CAST -Wno-unused-parameter -Wno-overloaded-virtual -Wno-missing-declarations -Werror -std=c++11 -mtune=generic -march=x86-64 -DCOMPILE_CORE3_TESTS
 AM_CXXFLAGS = $(DEFAULT_FLAGS) -O0
 ENGINE3_PATH = ../../../../MMOEngine
 ENGINE3_MOCK_PATH = ../../../utils
